services/core-web/src/components/noticeOfWork/NOWProgressActions.js

```diff
@@ -64,15 +64,15 @@ export class NOWProgressActions extends Component {
 
   handleProgress = (tab, trigger) => {
     if (trigger === "Complete") {
-      this.stopProgress(tab);
+      return this.stopProgress(tab);
```

Comment:
This whole thing seems like it would be simpler and more readable with an implicit return and ternary. Thoughts?

handleProgress = (tab, trigger) => trigger === "Complete" ? this.stopProgress(tab) : 
      this.startOrResumeProgress(tab, trigger);